### PR TITLE
refactor: improve the view of crm tables and pages

### PR DESCRIPTION
**Pull request by yusualhashash**
## How has this been tested?

- [ ] Prod
- [x] Development

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] Added all translations, no hard coded text. 
- [ ] Updated the UI (if needed).
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: apps/web/src/app/[lang]/app/[type]/crm/[domain]/[data]/(alternative)/details/[id]/form.tsx [hunk 1]
```diff
@@ -120,17 +119,16 @@ export default function Form({
     <div className="h-full overflow-hidden">
       <PageHeader
         LinkElement={Link}
-        description="Edit"
+        description={`Edit ${params.data}`}
         href={getBaseLink(`/app/admin/crm/${params.domain}/${params.data}`)}
-        title="Edit"
+        title={`Edit ${params.data}`}
```
Comment (a0m0rajab) [inline apps/web/src/app/[lang]/app/[type]/crm/[domain]/[data]/(alternative)/details/[id]/form.tsx:124]: Can we use the text from the language model that we have?

File: apps/web/src/app/[lang]/app/[type]/crm/[domain]/[data]/(alternative)/details/[id]/form.tsx [hunk 2]
```diff
@@ -120,17 +119,16 @@ export default function Form({
...
       />
-
       <SectionLayout
         sections={[
           { name: "Organization", id: "organization" },
-          { name: "Email", id: "email" },
           { name: "Telephone", id: "telephone" },
           { name: "Address", id: "address" },
+          { name: "Email", id: "email" },
```
Comment (a0m0rajab) [inline apps/web/src/app/[lang]/app/[type]/crm/[domain]/[data]/(alternative)/details/[id]/form.tsx:131]: we can get this from the language.

File: apps/web/src/app/[lang]/app/[type]/crm/[domain]/[data]/(alternative)/details/[id]/form.tsx [hunk 3]
```diff
@@ -211,6 +191,24 @@ export default function Form({
...
+              void handleSubmit(values, "email");
+            }}
+            values={{
+              emailAddress: emailInfo?.emailAddress,
+              typeCode: emailInfo?.typeCode?.toString(),
+              primaryFlag: emailInfo?.primaryFlag,
+            }}
+          >
+            <AutoFormSubmit className="float-right">
+              Save Changes
```
Comment (a0m0rajab) [inline apps/web/src/app/[lang]/app/[type]/crm/[domain]/[data]/(alternative)/details/[id]/form.tsx:208]: Let's change this as well

File: apps/web/src/app/[lang]/app/[type]/crm/[domain]/[data]/(alternative)/new/page.tsx [hunk 4]
```diff
@@ -67,9 +72,9 @@ export default function Page({
     <>
       <PageHeader
         LinkElement={Link}
-        description="Add New"
+        description={`Add New ${params.data}`}
         href={getBaseLink(`/app/admin/crm/${params.domain}/${params.data}`)}
-        title="Add New"
+        title={`Add New ${params.data}`}
```
Comment (a0m0rajab) [inline apps/web/src/app/[lang]/app/[type]/crm/[domain]/[data]/(alternative)/new/page.tsx:77]: Langauge

**Review by a0m0rajab — CHANGES_REQUESTED** [review]

File: apps/web/src/app/[lang]/app/[type]/crm/[domain]/[data]/(alternative)/details/[id]/form.tsx [hunk 5]
```diff
@@ -120,17 +130,24 @@ export default function Form({
     <div className="h-full overflow-hidden">
       <PageHeader
         LinkElement={Link}
-        description="Edit"
+        description={
+          languageData[
+            `${formData.title?.replaceAll(" ", "")}.Edit` as keyof typeof languageData
+          ]
+        }
```
Comment (a0m0rajab) [inline apps/web/src/app/[lang]/app/[type]/crm/[domain]/[data]/(alternative)/details/[id]/form.tsx:137]: for the edit and new, can we go with: 
```js
string = `${langaugeData["edit"]} ${langaugeData["Merchant"]}`
```

File: apps/web/src/app/[lang]/app/[type]/crm/[domain]/[data]/schemas.gen.ts
Comment (ertugrulcan-ays) [inline apps/web/src/app/[lang]/app/[type]/crm/[domain]/[data]/schemas.gen.ts:1]: manupulate backend schema instead of creating whole schema

**Review by ertugrulcan-ays — APPROVED** [review]

**Pull request merged**